soh/soh/Enhancements/randomizer/randomizer.cpp

```diff
@@ -326,7 +326,7 @@ void Randomizer::LoadMerchantMessages() {
     CustomMessageManager::Instance->CreateMessage(
         Randomizer::merchantMessageTableID, TEXT_BEAN_SALESMAN_BUY_FOR_100,
         CustomMessage("I never thought I'd say this, but I'm &selling the last %rMagic Bean%w. %r99%w Rupees...\x1B&%gYes&No%w",
-                      "Ich hätte nie gedacht, daß ich das&sage, aber ich verkaufe die&letzte %rWundererbse%w für %r99%w Rubine.\x1B&%gJa&Nein%w",
+                      "Ich hätte nie gedacht, daß ich das&sage, aber ich verkaufe die letzte^%rWundererbse%w für %r99%w Rubine.\x1B&%gJa&Nein%w",
```

Comment:
Because OoT is a well coded video game, this also breaks the yes no box, try adding an extra newline after Rubine.

Reply:
Could you explain this in detail? I would assume that the cursor for the binary selection is always in the third and the fourth line and therefore it is necessary to move to this position in the second box as well.

Reply:
Yes, this is what happens if you don't have 4 lines total in a yes/no box:
![image](https://github.com/user-attachments/assets/a0173e1a-20df-48ab-8b1d-f4c94743f60d)

Reply:
Thank you for the clarification!